source/common/upstream/health_checker_base_impl.cc

```diff
@@ -338,13 +338,14 @@ bool networkHealthCheckFailureType(envoy::data::core::v3::HealthCheckFailureType
 } // namespace
 
 HealthTransition HealthCheckerImplBase::ActiveHealthCheckSession::setUnhealthy(
-    envoy::data::core::v3::HealthCheckFailureType type) {
+    envoy::data::core::v3::HealthCheckFailureType type, bool retriable) {
```

Comment:
Compared to adding a boolean parameter, can we add a new HealthCheckFailure value `Retriable`?